lib/extend.es6

```diff
@@ -15,6 +15,7 @@ export default (options = {}) => {
         options.encoding = options.encoding || 'utf8';
         options.root = options.root || './';
         options.plugins = options.plugins || [];
+        options.strict = ('strict' in options) ? !!options.strict : true;
```

Comment:
may be better ?
```js
Object.prototype.hasOwnProperty.call(options, 'strict')
```

Reply:
The same distinction isn't made for the other options, but sure :-)